tests/python/pants_test/backend/python/tasks/test_ctypes.py

```diff
@@ -0,0 +1,117 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from builtins import str
+from textwrap import dedent
+
+from twitter.common.collections import OrderedDict
+
+from pants.backend.native.targets.native_artifact import NativeArtifact
+from pants.backend.native.targets.native_library import CLibrary, CppLibrary
+from pants.backend.native.tasks.c_compile import CCompile
+from pants.backend.native.tasks.cpp_compile import CppCompile
+from pants.backend.native.tasks.link_shared_libraries import LinkSharedLibraries
+from pants.backend.python.targets.python_distribution import PythonDistribution
+from pants_test.backend.python.tasks.python_task_test_base import check_wheel_platform_matches_host
+from pants_test.backend.python.tasks.util.build_local_dists_test_base import \
+  BuildLocalPythonDistributionsTestBase
+
+
+class TestBuildLocalDistsWithCtypesNativeSources(BuildLocalPythonDistributionsTestBase):
+
+  _extra_relevant_task_types = [CCompile, CppCompile, LinkSharedLibraries]
+
+  _dist_specs = OrderedDict([
+
+    ('src/python/plat_specific_c_dist:ctypes_c_library', {
+      'key': 'ctypes_c_library',
+      'target_type': CLibrary,
```

Comment:
Might be able to remove `target_type` key? I can't find any usages.

Reply:
It's in the superclass -- it's one of the kwargs that is forwarded to `self.make_target()`.

Reply:
👍

Reply:
(if that's too indirect let me know if there would be a useful comment to add!)